assets/app/features/upgrade/upgrade.module.js

```diff
@@ -2,5 +2,5 @@
     'use strict';
 
     angular
-        .module('crowbarApp.upgrade', []);
+        .module('crowbarApp.upgrade', ['ui.router', 'pascalprecht.translate', 'suseData']);
```

Comment:
Question: where is `pascalprecht.translate` coming from and being used for? Is this different than `$translate`?